src/__tests__/Accordion.spec.js

```diff
@@ -1,37 +1,148 @@
 import React from 'react';
-import { mount } from 'enzyme';
-import { Accordion } from '..';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Accordion, AccordionItem, AccordionHeader, AccordionBody } from '..';
+import { testForCustomClass, testForCustomTag } from '../testUtils';
+
+beforeEach(() => {
+  jest.useFakeTimers();
+});
+
+afterEach(() => {
+  jest.runOnlyPendingTimers();
+  jest.useRealTimers();
+});
 
 describe('Accordion', () => {
-  it('should render with "accordion" class', () => {
-    const wrapper = mount(
-      <Accordion open="this accordion" toggle={() => {}} />,
+  const accordionItems = [
+    <AccordionItem data-testid="item-1" key="1">
+      <AccordionHeader targetId="1" data-testid="head-1">
+        Accordion Item 1
+      </AccordionHeader>
+      <AccordionBody accordionId="1" data-testid="body-1">
+        This is the body of first item.{' '}
+      </AccordionBody>
+    </AccordionItem>,
+    <AccordionItem data-testid="item-2" key="2">
+      <AccordionHeader targetId="2" data-testid="head-2">
+        Accordion Item 2
+      </AccordionHeader>
+      <AccordionBody accordionId="2" data-testid="body-2">
+        This is the body of second item.{' '}
+      </AccordionBody>
+    </AccordionItem>,
+    <AccordionItem data-testid="item-3" key="3">
+      <AccordionHeader targetId="3" data-testid="head-3">
+        Accordion Item 3
+      </AccordionHeader>
+      <AccordionBody accordionId="3" data-testid="body-3">
+        This is the body of third item.{' '}
+      </AccordionBody>
+    </AccordionItem>,
+  ];
+
+  it('should render a div with "accordion" class', () => {
+    const { getByTestId } = render(
```

Comment:
Let's standardize on one way of getting these query functions (and other things). It seems like using `screen` from the import is the recommended way:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

So:
```javascript
import { render, screen } from '@testing-library/react';

.....

const node = screen.getByTestId('test');
```